src/jdk.compiler/share/classes/com/sun/tools/javac/processing/JavacProcessingEnvironment.java

```diff
@@ -300,10 +300,7 @@ private void initProcessorIterator(Iterable<? extends Processor> processors) {
             try {
                 processorIterator = List.of(new PrintingProcessor()).iterator();
             } catch (Throwable t) {
-                AssertionError assertError =
-                    new AssertionError("Problem instantiating PrintingProcessor.");
-                assertError.initCause(t);
-                throw assertError;
+                throw new AssertionError("Problem instantiating PrintingProcessor.", t);
```

Comment:
When this annotation processing code was written in JDK 6, I wondered why we didn't use the other constructor at the time. Turns out, that constructor of AssertionError was only added in JDK 7 under [JDK-6935997](https://bugs.openjdk.java.net/browse/JDK-6935997) :-)